ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -16,49 +16,59 @@
      */
     function renderLayer(data) {
 
+        // Decode Indices from data.visualization[].coordinate_indices
         var deserializedData = InternalBuildingRepresentation.read(new Pbf(data));
-        var coordsIndexList = deserializedData.visualization[0].coordinates;
-        var coordsRangeBuffer = coordsIndexList.buffer.slice(coordsIndexList.byteOffset, coordsIndexList.buffer.byteLength);
-        var coordsRange = new Uint32Array(coordsRangeBuffer);
-        for (var i = 0; i < coordsRange.length; i++) {
-            coordsRange[i] = swap32(coordsRange[i]);
+        var coordsIndexList, coordsRangeBuffer, coordsRange;
+        var coordsRangeList = [];
```

Comment:
nit: this is probably worth encapsulating in a helper function (L22:30)

there's logic here that's specific to the proto data structure that we want to abstract away from the user. Right now the output is just a list of THREE.Line objects, but in the future we might want to give the user more access to customize what they want to show and hide.